source/slang/slang-check-conformance.cpp

```diff
@@ -209,9 +210,12 @@ namespace Slang
 
                     if(_isDeclaredSubtype(originalSubType, inheritedType, superTypeDeclRef, outWitness, &breadcrumb))
                     {
-                        return true;
+                        found = true;
```

Comment:
It seems that we can no longer break out of the loop.

Reply:
Correct. The early exit was just an optimization, so in this case we are stuck looping over all extensions even if we find a hit early on. The asymptotic complexity of the code hasn't changed, though.

A future change could restore the optimization, at the cost of making the iteration routine a bit more complex.